### ISSUE TITLE
Enhance a11y for `Scroll`

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#1247]
path: stories/components/motion/scroll.stories.tsx

### Description
1 accessibility violation was found in the `Scroll`.

#### [useViewport](https://yamada-ui.github.io/yamada-ui/?path=/story/components-motion-scroll--use-viewport)

Expected the HTML found at $('.css-27mv67') to have no violations:
    <div class="css-27mv67"><div class="ui-motion css-1yr9jgb" style="opacity: 0;">Motion</div></div>

    Received:
    Scrollable region must have keyboard access (scrollable-region-focusable)

    Fix any of the following:
      Element should have focusable content
      Element should be focusable

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.9/scrollable-region-focusable?application=axeAPI

### Problem Statement/Justification
A11y is essential to ensure quality when operating a package.

### Additional Information
While it's ideal to cover all jobs, if there are too many jobs to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.
